lib/components/Utilities/Icon/index.tsx

```diff
@@ -10,16 +10,13 @@ import {
 const iconVariants = cva("inline-block shrink-0", {
   variants: {
     size: {
-      xl: "h-12 w-12",
-      lg: "h-8 w-8",
-      md: "h-5 w-5",
-      sm: "h-4 w-4",
-      xs: "h-3 w-3",
+      lg: "w-[24px]",
+      md: "w-[20px]",
```

Comment:
If you look at [spacing.ts](https://github.com/factorialco/factorial-one/blob/main/tailwind/spacing.ts), you can see how 24px would be 6, 20px is 5 and 16px is 4.

I think you may be getting other values with the hover preview of Tailwind IntelliSense (happens to me too), but those values are not the real ones.

Reply:
Ohh really? I'm gonna double-check, thank you!